blazor/diagram/Rulers.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Ruler settings in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about the Ruler feature in Syncfusion Blazor Diagram component and more.
+platform: Blazor
+control: Diagram Component
+documentation: ug
+---
+
+# Ruler Settings in Blazor Diagram Component
+
+The Ruler provides horizontal and vertical guides for measuring in the Diagram control. It can be used to measure diagram objects, indicate positions, and align diagram elements, making it especially useful in creating scale models.
+
+### Adding Rulers to the Diagram Component
+
+- The `RulerSettings` property is used to control the visibility and appearance of the ruler in the diagram.
+
+The following code demonstrates how to add a ruler to the diagram.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+
+<SfDiagramComponent @ref="@diagram" Height="600px" >
+	<RulerSettings>
+        <HorizontalRuler>
+          </HorizontalRuler>
+        <VerticalRuler >
+          </VerticalRuler>
+    </RulerSettings>
+</SfDiagramComponent>
+
+```
+
+![Ruler](images/Ruler.png)
+
+## Customizing the Ruler
+
+- The `RulerSettings` `HorizontalRuler` and `VerticalRuler` properties are used to customize the appearance of the rulers in the diagram.
```

Comment:
The  `HorizontalRuler` and `VerticalRuler` properties `RulerSettings` class are used to customize the appearance of the rulers in the diagram. The following properties are used to customize the appearance of both the horizontal and vertical rulers.